# Goal definition
float64[] positions
float64[] timestamps
int64 test_no

---
# Result definition
bool success

---
# Feedback definition
float64 current_position
float64 current_timestamp
